lib/http.js

```diff
@@ -748,6 +750,69 @@ async function _generateRootCapability(url) {
   };
 }
 
+/**
+ * Gets the expectedRootCapability for a zCap from the invocationTarget.
+ *
+ * @param {object} Options to use.
+ * @param {string|Array<string>} options.invocationTarget - The
+ *   invocationTarget.
+ * @param {string} options.edvId - The edvId.
+ * @param {string} [options.docPath] - An optional docPath.
+ *
+ * @return {string|Array<string>} The expectedRootCapability for
+ *   the invocationTarget.
+ */
+function _getExpectedRootCapability({invocationTarget = '', edvId, docPath}) {
+  const verifyDocPath = url =>
+    url.includes('documents') && !url.endsWith('documents');
+  if(typeof invocationTarget === 'string') {
+    // this handles calls on the authorizations api
+    if(invocationTarget.endsWith('authorizations')) {
+      return `${edvId}/zcaps/authorizations`;
+    }
+    // this handles calls on the documents api
+    if(invocationTarget.endsWith('documents')) {
+      return `${edvId}/zcaps/documents`;
+    }
+    // chunks and documents by id have the same rootCap
+    if(verifyDocPath(invocationTarget)) {
+      return [
+        `${edvId}/zcaps${docPath}`,
+        `${edvId}/zcaps/documents`
+      ];
+    }
+  }
+  if(Array.isArray(invocationTarget)) {
```

Comment:
`invocationTarget` should not be an array. We should remove this ... and see out if it's ever being called like that and figure out what's going on.